src/compiler/crystal/command.cr

```diff
@@ -52,9 +52,9 @@ class Crystal::Command
       when "init".starts_with?(command)
         options.shift
         init
-      when "build".starts_with?(command)
+      when "build".starts_with?(command), "compile".starts_with?(command)
         options.shift
-        build
+        compile command
```

Comment:
I think the usage saying "compile" when "build" is still used would actually be a good thing :)